gen-lamp-inputs.py

```diff
@@ -18,7 +18,7 @@
         runname = '%s-%s' % (gcm, scen)
 
         cfgfile = 'lamp-inputs/%s.cfg' % runname
-        print 'cfgfile = %s' % cfgfile
+        print('cfgfile = %s' % cfgfile)
```

Comment:
I think we can probably delete this file.  I believe it was generating inputs for a particular set of runs that somebody wanted for the LAMP project.